README.md

```diff
@@ -9,10 +9,17 @@ You can submit issues and pull requests to extend functionality or fix potential
 
 *Please note:* this repository is not an official support channel. To get project-specific help, please create support ticket through [Support Portal](https://support.magento.com). Support-related issues will be closed with the request to open a support ticket.
 
+## Magento Cloud Module for Core
+The ece-tools  package uses extended core functionality that is provided by the [Magento Cloud Components](https://github.com/magento/magento-cloud-components) module. Starting with ece-tools `2002.0.20`, this module is required to support some advanced features, like cache warm-up using regex. The [Magento Cloud Components](https://github.com/magento/magento-cloud-components) module is installed automatically when you upgrade the ece-tools package to `2002.0.20` or later.
```

Comment:
The use of "like" means similar instead using "such as", which indicates it is a real example.

```suggestion
The ece-tools  package uses extended core functionality that is provided by the [Magento Cloud Components](https://github.com/magento/magento-cloud-components) module. Starting with ece-tools `2002.0.20`, this module is required to support some advanced features, such as cache warm-up using regex. The [Magento Cloud Components](https://github.com/magento/magento-cloud-components) module is installed automatically when you upgrade the ece-tools package to `2002.0.20` or later.
```